integration.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Compiling the webserver
+echo "Building webserver."
+make
+
+# starting webserver
+{	
+	echo "Starting webserver."
+	timeout 10 ./webserver test_config_1
+	echo "Stopping webserver."
+}&
+sleep 3
+
+# testing webserver
+echo "Running command curl -D headers.txt http://localhost:12345 > response.txt"
+curl -D headers.txt http://localhost:12345 > response.txt
+
+# checking reponse
+echo "Checking response."
+if grep -q "HTTP/1.1 200 OK" headers.txt;then
+	if grep -q "Content-Type: text/plain" headers.txt;then
+		#Checking if content-length is correct
+		count=`wc -c response.txt | awk '{print $1}'`
+		if grep -q "Content-Length: $count" headers.txt;then
+			echo "All response headers are correct."
+		else
+			echo "HTTP response does not contain correct Content-Length."
+			exit 12
```

Comment:
are those error codes made up or official http... official http would be good but if these dont exist then maybe document error codes. not time-sensitive though